### PR TITLE
Update DeepLabv3+ Baseline Hyperparameters

### PR DESCRIPTION
**Pull request by Landanjs**
I swear this is the last time. Last week, it was noticed the deeplabv3+ could be trained on 4x the batch size while still maintaining the same performance across SSRs. This gives a modest reduction in training time on a single node, but could pretty impactful in multi-node settings. This also lets us remove sync batch norm.

Wandb for SSR=1: https://wandb.ai/mosaic-ml/landan-dlv3p-baseline/groups/baseline-bs128-ssr1.0/workspace?workspace=user-landanjs

### CODE REVIEW

**Comment by abhi-mosaic** [comment]
Big fan of bigger batch sizes :) Maybe we can collect some multi-node benchmarks soon.

**Review by coryMosaicML — APPROVED** [review]
Looks good to me!
